webapp/portlet/src/main/webapp/space-members/components/SpaceInvitationDrawer.vue

```diff
@@ -161,6 +163,16 @@ export default {
     },
     defaultAvatar() {
       return `${eXo.env.portal.context}/${eXo.env.portal.rest}/v1/social/users/default-image/avatar`;
+    },
+
+    invitedUser() {
+      if ( this.invitedMembers.length === 1){
+        return this.invitedMembers[0].remoteId;
+      }
+      else
+      {
+        return 0;
+      }
```

Comment:
Return the full name instead of the username (remoteId)
Try to make it a one line statement

Reply:
ok, noted